Dockerfile

```diff
@@ -10,8 +10,9 @@ RUN cargo build --release
 
 FROM alpine:3.20
 
-WORKDIR /usr/src/indexer
+WORKDIR /usr/src
 
-COPY --from=builder /usr/src/indexer/target/release/indexer .
+COPY --from=builder /usr/src/target/release/indexer .
+COPY --from=builder /usr/src/target/release/cli .
```

Comment:
Does cli exist?

Reply:
Not anymore indeed, [done in e2cad1f](https://github.com/K-Kluster/kasia-indexer/pull/10/commits/e2cad1f9ce2f6dc378ac2f7af54b6941a562e467)

By the way I'm currently creating the CI for automatically build the docker image & publish to docker hub